packages/terra-consumer-nav/src/Nav.scss

```diff
@@ -27,6 +27,7 @@
     background: var(--terra-consumer-nav-background, rgba(255, 255, 255, 0.25));
     color: var(--terra-consumer-nav-text-color);
     height: 100%;
```

Comment:
shoudln't height and min-height be the same unit value?

Reply:
since nav is relative positioned, (height:100vh) gives nav a fixed height equal to the window height. it doesn't work when the body content is longer than window height. see screenshot:
<img width="850" alt="screen shot 2017-09-06 at 3 18 35 pm" src="https://user-images.githubusercontent.com/5943136/30132643-b207b51c-9316-11e7-914b-8a573f50c0cc.png">

Reply:
ok thanks that makes sense. Maybe leave that as a comment in the code